### PR TITLE
Add EOF Errors to evmone configuration

### PR DESCRIPTION
**Pull request by hugo-dc**

### CODE REVIEW

File: retesteth/configs/clientconfigs/evmone.cpp [hunk 1]
```diff
@@ -274,42 +275,66 @@ string const evmone_config = R"({
...
-    evmone -v
+    evmone-t8n -v
+elif [ $1 = "eof" ]; then
+    result=$(echo $5 | evmone-eofparse)
+    echo $result | grep "OK" > /dev/null
+    if [ $? -eq 0 ]; then
+      echo "ok."
+    else
+      echo "$result"
+    fi
```
Comment (gumb0) [inline retesteth/configs/clientconfigs/evmone.cpp:337]: Note @winsvega : this hack is a workaround for this issue https://github.com/ethereum/retesteth/issues/223

Comment (gumb0) [reply retesteth/configs/clientconfigs/evmone.cpp:337]: But we'd be fine with merging it with this workaround for now.

**Review by gumb0 — APPROVED** [review]
Error messages and start script look fine to me.

File: retesteth/configs/clientconfigs/evmone.cpp [hunk 2]
```diff
@@ -274,42 +275,66 @@ string const evmone_config = R"({
...
 if [ -z $wevm ]; then
-   >&2 echo "Can't find EvmOne's 'evmone' executable alias in the system path!"
+   >&2 echo "Can't find EvmOne's 'evmone-t8n' executable alias in the system path!"
    exit 1
 fi
 
-if [ $1 = "eof" ] || [ $1 = "t8n" ] || [ $1 = "b11r" ]; then
-    evmone $1 $2 $3 $4 $5 $6 $7 $8 $9 $10 $11 $12 $13 $14 $15 $16 $17 $18 $19 $20 $21 $22 $23 $24 $25 $26
+if [ $1 = "t8n" ] || [ $1 = "b11r" ]; then
+    evmone-t8n $2 $3 $4 $5 $6 $7 $8 $9 $10 $11 $12 $13 $14 $15 $16 $17 $18 $19 $20 $21 $22 $23 $24 $25 $26
```
Comment (winsvega) [inline retesteth/configs/clientconfigs/evmone.cpp:327]: Need to check if it still works with .py tests.

File: retesteth/configs/clientconfigs/evmone.cpp [hunk 3]
```diff
@@ -274,42 +275,66 @@ string const evmone_config = R"({
...
 elif [ $1 = "-v" ]; then
-    evmone -v
+    evmone-t8n -v
+elif [ $1 = "eof" ]; then
+    result=$(echo $5 | evmone-eofparse)
```
Comment (winsvega) [inline retesteth/configs/clientconfigs/evmone.cpp:331]: Passing number 5 here is bot reliable. Its position might change. What are the 2, 3, 4 that ignored?

File: retesteth/configs/clientconfigs/evmone.cpp (hunk 1 above)
Comment (winsvega) [reply retesteth/configs/clientconfigs/evmone.cpp:337]: The response can verify in eof validation code yes

**Pull request merged**